### ISSUE TITLE
Compiling error

### ISSUE THREAD

**Issue by sausix** [bartoszek/AUR-blender-git#2]
Sorry.
Doesn't compile anymore.

```
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
OPENEXR_HALF_LIBRARY (ADVANCED)
    linked by target "cycles_kernel_osl" in directory /home/as/.cache/yay/blender-git/src/blender/intern/cycles/kernel/osl
    linked by target "cycles_render" in directory /home/as/.cache/yay/blender-git/src/blender/intern/cycles/render
OPENEXR_ILMIMF_LIBRARY (ADVANCED)
    linked by target "cycles_kernel_osl" in directory /home/as/.cache/yay/blender-git/src/blender/intern/cycles/kernel/osl
    linked by target "cycles_render" in directory /home/as/.cache/yay/blender-git/src/blender/intern/cycles/render
```

Affects yay, clean builds and makepkg.

Other issues with our qfix friends:

**openimageio-qfix**:
```
[154/241] Building CXX object src/libOpenImageIO/CMakeFiles/OpenImageIO.dir/__/openexr.imageio/exrinput.cpp.o
FAILED: src/libOpenImageIO/CMakeFiles/OpenImageIO.dir/__/openexr.imageio/exrinput.cpp.o 
/usr/bin/c++ -DEMBED_PLUGINS=1 -DOpenImageIO_EXPORTS -DUSE_BOOST_ASIO=1 -DUSE_EXTERNAL_PUGIXML=1 -DUSE_FFMPEG -DUSE_FREETYPE=1 -DUSE_GIF -DUSE_HEIF=1 -DUSE_JPEG_TURBO=1 -DUSE_LIBRAW=1 -DUSE_OCIO=1 -DUSE_OPENCOLORIO=1 -DUSE_OPENCV=1 -DUSE_OPENJPEG -DUSE_OPENVDB=1 -DUSE_PTEX -DUSE_STD_REGEX -DUSE_TBB=1 -DUSE_WEBP=1 -D__STDC_CONSTANT_MACROS -D__STDC_LIMIT_MACROS -Iinclude/OpenImageIO -Iinclude -Isrc/include -I../src/include -I../src/dds.imageio/squish -I/usr/include/openjpeg-2.4 -I/OpenEXR -I/opt/oiio/include -I/usr/include/opencv4 -I/usr/include/freetype2 -isystem /usr/include/OpenEXR -isystem /usr/include/Imath -isystem /opt/ocio/include -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -fexceptions         -Wp,-D_FORTIFY_SOURCE=2,-D_GLIBCXX_ASSERTIONS         -Wformat -Werror=format-security         -fstack-clash-protection -fcf-protection -O3 -DNDEBUG -fPIC -Wall -Wno-unused-local-typedefs -Wno-unused-result -Wno-aligned-new -Wno-noexcept-type -fno-math-errno -std=c++14 -MD -MT src/libOpenImageIO/CMakeFiles/OpenImageIO.dir/__/openexr.imageio/exrinput.cpp.o -MF src/libOpenImageIO/CMakeFiles/OpenImageIO.dir/__/openexr.imageio/exrinput.cpp.o.d -o src/libOpenImageIO/CMakeFiles/OpenImageIO.dir/__/openexr.imageio/exrinput.cpp.o -c ../src/openexr.imageio/exrinput.cpp
../src/openexr.imageio/exrinput.cpp:115:20: error: 'Int64' in namespace 'Imath' does not name a type
```

**openshadinglanguage-qfix**:
```
[15/129] Generating llvm_ops.bc.cpp
FAILED: src/liboslexec/llvm_ops.bc.cpp 
cd /home/as/.cache/yay/openshadinglanguage-qfix/src/OpenShadingLanguage-Release-1.11.11.0/src/liboslexec && /usr/bin/clang++ -DOSL_FAST_MATH=1 -D__STDC_LIMIT_MACROS -D__STDC_CONSTANT_MACROS -DUSE_STD_REGEX -DOSL_LLVM_VERSION=111 -DOSL_LLVM_FULL_VERSION="11.1.0" -std=c++14 -I/home/as/.cache/yay/openshadinglanguage-qfix/src/OpenShadingLanguage-Release-1.11.11.0/src/liboslexec -I/home/as/.cache/yay/openshadinglanguage-qfix/src/OpenShadingLanguage-Release-1.11.11.0/src/include -I/home/as/.cache/yay/openshadinglanguage-qfix/src/OpenShadingLanguage-Release-1.11.11.0/build/include -I/opt/oiio/include -I/usr/include -I/usr/include -isystem\ /usr/include -DOSL_COMPILING_TO_BITCODE=1 -Wno-deprecated-register -Wno-ignored-attributes -Wno-unknown-attributes -O3 -fno-math-errno -S -emit-llvm -o /home/as/.cache/yay/openshadinglanguage-qfix/src/OpenShadingLanguage-Release-1.11.11.0/build/src/liboslexec/llvm_ops.s llvm_ops.cpp && /usr/bin/llvm-as -f -o /home/as/.cache/yay/openshadinglanguage-qfix/src/OpenShadingLanguage-Release-1.11.11.0/build/src/liboslexec/llvm_ops.bc /home/as/.cache/yay/openshadinglanguage-qfix/src/OpenShadingLanguage-Release-1.11.11.0/build/src/liboslexec/llvm_ops.s && python /home/as/.cache/yay/openshadinglanguage-qfix/src/OpenShadingLanguage-Release-1.11.11.0/src/liboslexec/serialize-bc.py /home/as/.cache/yay/openshadinglanguage-qfix/src/OpenShadingLanguage-Release-1.11.11.0/build/src/liboslexec/llvm_ops.bc /home/as/.cache/yay/openshadinglanguage-qfix/src/OpenShadingLanguage-Release-1.11.11.0/build/src/liboslexec/llvm_ops.bc.cpp osl_llvm_compiled_ops
In file included from llvm_ops.cpp:80:
/home/as/.cache/yay/openshadinglanguage-qfix/src/OpenShadingLanguage-Release-1.11.11.0/build/include/OSL/oslconfig.h:19:13: fatal error: 'Imath/ImathLimits.h' file not found
```

**Comment by bartoszek** [comment]
Thanks for poking me about it. I saw a patch for `openexr:3` in `community/blender` will merger it tomorrow...

**Comment by bartoszek** [comment]
Resolved f755cc4b68579241fcc31feaafbd4a8097280e95

**Comment by sausix** [comment]
Thanks again bartoszek!

Some warnings, but it compiles and runs again.
